src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "Root1234");
```

Comment:
Hardcoding database credentials (username and password) in the source code is a security risk. Consider externalizing these credentials to a configuration file or environment variables to enhance security.